### PR TITLE
All Assignments done

### PR DESCRIPTION
**Pull request by AbhishekParida05**

### CODE REVIEW

**Comment by AbhishekParida05** [comment]
done

**Review by AbhishekParida05 — COMMENTED** [review]
self test

**Comment by AbhishekParida05** [comment]
Done assignment

**Comment by AbhishekParida05** [comment]
Done Assignment

**Comment by AbhishekParida05** [comment]
Done Assignment

File: assignment.js
Comment (NaveenMohanty) [inline assignment.js:1]: Delete this file

**Review by NaveenMohanty — CHANGES_REQUESTED** [review]
Do the mentioned change and commit and push the code in this branch

**Review by NaveenMohanty — APPROVED** [review]

**Pull request merged**